App/templates/App/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title> USER CONTRIBUTION SUMMARY </title>
+</head>
+<body>
+		<center>
+	     <form method="post"> 
+	     	{% csrf_token %}
+	     	Enter Username: <input type="text" name="username">
+	     	<br>
+	     	<br>
+	     	<button type='submit'> Submit </button>
+	     </form>
+	    </center> 
+	    
+	    <h2>
+	    {% if userid %}
+	    	Username = {{username}} <br>
+	    	Userid = {{userid}} <br>
+	    	<a href="https://phabricator.wikimedia.org/p/{{username}}/"> Profile link </a>
```

Comment:
The Phabricator username is not typically the same as the wiki username.

Reply:
Alright. I added that link to augment to the information. Since I just got to know that phabricator and wiki usernames are not same, I'd rather remove this link. Thank you .